app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -166,6 +176,40 @@ public Study getStudy(StudyIdentifier studyId) {
         return studyDao.getStudies();
     }
 
+    public Study createStudyAndUser(StudyAndUserHolder studyAndUserHolder) throws SynapseException {
+        checkNotNull(studyAndUserHolder, Validate.CANNOT_BE_NULL, "study and users");
+        if (studyAndUserHolder.getUsers() == null) {
+            throw new BadRequestException("User list cannot be null.");
+        }
+        if (studyAndUserHolder.getStudy() == null) {
+            throw new BadRequestException("Study cannot be null.");
+        }
```

Comment:
I agree with this, also it can help prevent partially creating the synapse entities only to receive an error, which will leave things broken.